specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -1010,29 +1010,29 @@
           "items": {
             "$ref": "#/definitions/Operation"
           },
-          "description": "List of Microsoft.ManagedServices operations."
+          "description": "The list of Microsoft.ManagedServices operations."
         }
       },
       "type": "object",
-      "description": "List of the operations."
+      "description": "The list of the operations."
     },
     "Authorization": {
       "properties": {
         "principalId": {
           "type": "string",
-          "description": "Principal Id of the security group/service principal/user that would be assigned permissions to the projected subscription"
+          "description": "The identifier of the principal."
```

Comment:
>The identifier of the principal. [](http://example.com/codeflow?start=26&length=32)

I am thinking should be mention about AAD principal or something to convey it is AAD objectId.